RELEASE_NOTES

```diff
@@ -1,6 +1,37 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.6.0 (18 September 2024)
+--------------------------
+
+This is a minor release of the EESSI build-and-deploy bot.
+
+Bug fixes:
+* mark 'with_accelerator' setting as required (#282)
```

Comment:
I would collapse this with the #280 below, since it's not a bug fix compared to v0.5.0